src/main/groovy/com/github/jrubygradle/GemUtils.groovy

```diff
@@ -94,10 +94,15 @@ class GemUtils {
                 gemsToProcess.each { File gem ->
                     args gem
                 }
+                // there are a few extra args which look like defaults
+                // but we need to make sure any config in $HOME/.gemrc
+                // is overwritten
                 args '--ignore-dependencies',
                      "--install-dir=${destDir.absolutePath}",
-                     '-N',
-                     '--platform=java'
+		     '--no-user-install',
```

Comment:
These changes are in #108 already and has been merged, so can be removed from this PR.